docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     volumes:
     - pgdata:/var/lib/postgresql/data
   clamav:
-    image: ghcr.io/ministryofjustice/hmpps-clamav:sha-0ae98e9
+    image: ghcr.io/ministryofjustice/hmpps-clamav:sha-ae9a953
```

Comment:
Maybe we should revert to using `:latest`
Historically we used it, but HMPPS broke their build pipeline and took a few days to recover it.  We locked at a known SHA so we could still run tests and deploy.
Using latest pros:
* up-to-date build
Cons
* fragile if build pipeline builds again

Using locked sha pros
* known good build

Cons
* quickly out of date
* requires 'regular' updates

Reply:
🤦‍♂️ Just read that jira ticket!  This looks fine 🙂